lib/ec2.ex

```diff
@@ -56,4 +56,13 @@ defmodule AntikytheraAws.Ec2.ClusterConfiguration do
     %Httpc.Response{body: body} = Httpc.get!(@availability_zone_metadata_url)
     body
   end
+
+  @impl true
+  defun health_check_grace_period_in_seconds() :: pos_integer do
+    run_cli(["autoscaling", "describe-auto-scaling-groups", "--auto-scaling-group-names", @auto_scaling_group_name], fn j ->
```

Comment:
`run_cli/2` returns `{:error, :script_error}`, so the return type of this function is wrong.
It would be better to return a default value and write an error log in the `:error` case.

Reply:
Thanks! I want to use 300 as the default value because it is one in [AWS ASG](https://docs.aws.amazon.com/autoscaling/ec2/userguide/healthcheck.html#health-check-grace-period) and basically an a9a node needs some grace period.
Fixed in https://github.com/access-company/antikythera_aws/pull/8/commits/9f71a572c110d814bb0856eb3362b74a2adc1edf